Added Elks46PhoneCallAgent

AntonMyr opened the pull request:
# 46elks Phone Call agent
## What is this?
This is an agent that allows you to make phone calls with the help of https://46elks.com.

## What is covered so far?
The agent is done functionality wise.

The tests are very basic, they cover the validation and receive methods.

dsander commented:
Wow, they have quite the collection of services 😄 

Instead of repeating myself again, feel free to ping me of you have any issues creating the Agent gem for your three Agents.

AntonMyr commented:
@dsander Hey thank you for your feedback on my pull requests. I don't quite understand how to add the agent gems.

I started building my own and added it to my .env file in my local setup like this:
`ADDITIONAL_GEMS=huginn_awesome_agent(path: /local/path/to/huginn_awesome_agent)`

After I added it to the .env I also ran `bundle`.

Is the agent supposed to show up when you press "New Agent" because it doesn't? 

I tried with the (TodoistAgent)[https://github.com/stesie/huginn_todoist_agent] as well but that doesn't show up either. Any ideas what I might be doing wrong?

dsander commented:
I think you hit a bug that got reported recently. Agent gems seem not to work in development mode at the moment. Until we have that fixed you can work around it by making [this change](https://github.com/huginn/huginn/issues/2845#issuecomment-662127940).
